generator-prisma-client/react-router-starter-cloudflare-workerd/README.md

```diff
@@ -0,0 +1,118 @@
+  - Your **Prisma Postgres direct TCP connection string** which you will use with Prisma Client.
+    Learn more in the [docs](https://www.prisma.io/docs/postgres/database/direct-connections).
+
+## Tech Stack
+
+- React Router 7
+  - Runtime: Cloudflare Workerd
+  - Bundler: Vite 6
+- ESM
+  - `package.json` contains `{ "type": "module" }`
+- Prisma Client with the `prisma-client` generator
+  See the [Prisma schema file](./prisma/schema.prisma) for details.
+  
+  ```prisma
+  generator client {
+    provider = "prisma-client"
+    output = "../app/generated/prisma"
+    previewFeatures = ["driverAdapters", "queryCompiler"]
+    runtime = "workerd"
+  }
+  ```
+
+## Getting started
+
+### 1. Clone the repository
+
+Clone the repository, navigate into it and install dependencies:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+cd prisma-examples/generator-prisma-client/react-router-starter-cloudflare-workerd
+pnpm install
+```
+
+### 2. Configure environment variables
+
+Create a `.env` in the root of the project directory:
+
+```bash
+touch .env
+```
+
+Now, open the `.env` file and set the `DATABASE_URL` environment variables with the values of your connection string and your Prisma Postgres connection string:
+
+```bash
+# .env
+
+# Prisma Postgres connection string (used for migrations)
+DATABASE_URL="__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__"
+```
+
+Create a `.dev.vars` file in the root of the project directory:
+
+```bash
+touch .dev.vars
+```
+
+Now, open the `.dev.vars` file and set the `DIRECT_URL` environment variable with the value of your Prisma Postgres direct TCP connection string:
+
+```bash
+# .dev.vars
+
+# Postgres connection string (used for queries by Prisma Client)
+DIRECT_URL="__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__"
+```
+
+Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres + Accelerate connection string. Notice that the Accelerate connection string has the following structure: `prisma+postgres://accelerate.prisma-data.net/?api_key=<api_key_value>`.
+
+Note that `__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres direct TCP connection string. The direct connection string has the following structure: `postgres://<username>:<password>@<host>:<port>/<database>`.
+
+### 3. Run a migration to create the database structure and seed the database
+
+The [Prisma schema file](./prisma/schema.prisma) contains a single `Quotes` model and a `QuoteKind` enum. You can map this model to the database and create the corresponding `Quotes` table using the following command:
+
+```
+pnpm prisma migrate dev --name init
```

Comment:
ran into an error here, seems like `DIRECT_URL` isn't properly picked up from `.dev.vars` when running this command: 

```
$ pnpm prisma migrate dev --name init
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: Prisma schema validation - (get-config wasm)
Error code: P1012
error: Environment variable not found: DIRECT_URL.
  -->  prisma/schema.prisma:11
   | 
10 |   url       = env("DATABASE_URL")
11 |   directUrl = env("DIRECT_URL")
   | 

Validation Error Count: 1
[Context: getConfig]

Prisma CLI Version : 6.11.0
```